wiki/Offset/Universal_offset/en.md

```diff
@@ -1,12 +1,31 @@
 ---
-stub: true
+tags:
+  - UO
+  - global offset
 ---
 
 # Universal offset
 
 *For other uses, see [Offset](/wiki/Offset).*\
 *For step-by-step instructions on how to set the universal offset to the correct value, see [How to use the Offset Wizard](/wiki/Guides/How_to_Use_the_Offset_Wizard).*
 
-**Universal offset** is the audio offset [option](/wiki/Client/Options) that applies to all [beatmaps](/wiki/Beatmap) in the osu! client. Setting it to a value other than `0` adds a delay between every beatmap's song and the other associated sounds and graphics. Setting the universal offset to a positive value will make the sounds and graphics associated with the beatmap appear earlier, while setting it to a negative value will make them appear later.
+**Universal offset** is an [option](/wiki/Client/Options) that applies to audio files of all [beatmaps](/wiki/Beatmap) in the osu! client.
 
-Each player's optimal universal offset value differs due to variations in their systems. A Universal Offset, other than `0`, should only be used when **all beatmaps** have a noticeable delay. Otherwise, a [local offset](/wiki/Glossary/Local_song_offset) should be used instead. An incorrectly determined universal offset will cause significant timing problems on all beatmaps.
+## Behaviour
+
+Universal offset works by adding a specified delay between every beatmap's song and the other associated sounds and graphics. Since it is applied to the audio rather than beatmap objects, it produces an effect opposite than these of [local](/wiki/Offset/Local_offset) or [online](/wiki/Offset/Online_offset) offsets:
+
+- Positive values will move gameplay elements **earlier**
+- Negative values will move gameplay elements **later**
+
+It is usually worth keeping the universal offset at its default value, `0`, since an incorrectly determined universal offset will cause significant timing problems on all beatmaps. However, if **all beatmaps** have a consistent and noticeable delay, using a different value may be beneficial.[^local-offset] Each player's optimal universal offset value differs due to variations in their systems.
```

Comment:
```suggestion
It is usually worth keeping the universal offset at its default value, `0`, since an incorrectly determined universal offset will cause significant timing problems on all beatmaps. However, if **all beatmaps** have a consistent and noticeable timing issues, using a different value may be beneficial.[^local-offset] Each player's optimal universal offset value differs due to variations in their systems.
```